packages/create-rise/template-react-navigation/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "template-react-navigation",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "rise start",
+    "eject": "rise eject"
```

Comment:
We will have to remove this in favour of `tsx` since we don't really have CLI shipped yet. This is also non-blocking and I will just approve, we can handle that in a follow-up.